docs/guides/quick-start.md

```diff
@@ -15,7 +15,8 @@ pip install frictionless[sql] # to install a core plugin (optional)
 
 The framework supports CSV, Excel, and JSON formats by default. Please use the command above to install a core plugin and add support for SQL, Pandas, HTML, and others (check the [list of Frictionless Framework plugins and their status](https://framework.frictionlessdata.io/docs/references/plugins-reference)). Usually, you don't need to think about it in advance–frictionless will display a useful error message about a missing plugin with installation instructions.
 
-### Installation Troubleshooting
+### Troubleshooting
```

Comment:
Shorten it as it's already under the "Installation" section hierarchically?

I also got problems with some other heading as it seems they to be too long for the table of contents:

![toc](https://user-images.githubusercontent.com/557395/108033368-f676cd00-7044-11eb-9d35-d6df8f5d32ae.png)

Is it possible to shorten them too?

I think for highlighting text visually we can use other methods as in my opinion the main role of headings is to provide semantics (e.g. consider a blind person trying to navigate through them)

WDYT?